samples/nrf5340/netboot/README.rst

```diff
@@ -1,88 +1,95 @@
-Note that the build system will automatically enable the :option:`CONFIG_SECURE_BOOT` option for the network core application if the application on the application core has enabled :option:`CONFIG_BOOTLOADER_MCUBOOT` and :option:`CONFIG_BT_RPMSG_NRF53`.
+* If you don't need the capability to update the sample, enable the :option:`CONFIG_SECURE_BOOT` option in the application that runs on the network core.
+* If you need the capability to update the sample, enable the :option:`CONFIG_BOOTLOADER_MCUBOOT` option in the application that runs on the application core to add MCUboot to the build.
+  When doing so, the build system includes the sample in the build by enabling automatically the :option:`CONFIG_SECURE_BOOT` option for the application that runs on the network core.
+  In addition, you must also enable the :ref:`subsys_pcd` library for MCUboot by setting the :option:`CONFIG_PCD` option when building its image.
+  See :ref:`ug_multi_image` for more information about setting configurations to a specific image.
 
-A new set of firmware update files is generated by the build system when this image is included in a multi-image build.
-These files are identical to those described in :ref:`mcuboot:mcuboot_ncs`, except that they contain the network core application firmware and are prefixed with ``net_core_``.
+  .. note::
+     In this setup, MCUboot verifies any new image received through a device firmware update (DFU) transport layer, like a serial or a Bluetooth LE connection, for both the application and the network core.
+     This includes the network core bootloader image.
+     See :ref:`subsys_pcd` for more details.
 
-Testing
-=======
+When you include this sample in a multi-image build, the build system generates a new set of firmware update files.
+These files match the ones described in :ref:`mcuboot:mcuboot_ncs`, except that they contain the network core application firmware and are prefixed with ``net_core_``.
 
-To test the network core bootloader sample run the following commands:
+This sample can be found under :file:`samples/nrf5340/netboot/` in the |NCS| folder structure.
 
-#. |connect_terminal|
+See :ref:`configure_application` for information on how to enable the required configuration options.
+Then follow the instructions in :ref:`ug_nrf5340_building` to build and program the images for the network and application core.
 
-   Note that on the nRF5340 DK has multiple UART instances, so the correct port must be identified.
+.. note::
+   To try out the network core bootloader sample, you can use the :ref:`peripheral_uart` sample as the basis for the multi-image build.
+   This sample automatically includes the network core sample :ref:`bluetooth-hci-rpmsg-sample` when built for the nRF5340 DK.
+   You can then apply the options mentioned above to include the network core sample with or without MCUboot.
```

Comment:
The sample is only included if MCUboot is enabled for the `peripheral_uart` sample.

Reply:
The `peripheral_uart` is the only sample that the netboot sample can be applied with?

Reply:
No, my point was that for any app core sample you have to enable MCUboot for the netboot bootloader to be included.
In addition to this any app core sample must also include a net core child image (which `peripheral_uart` does)